server/graphql/v2/interface/Transaction.js

```diff
@@ -80,6 +80,9 @@ export const Transaction = new GraphQLInterfaceType({
       toAccount: {
         type: Account,
       },
+      virtualCardEmitterAccount: {
```

Comment:
In https://github.com/opencollective/opencollective-api/blob/f65084f876e15e3854d83745bd3f9b196af08a61/server/graphql/v2/enum/PaymentMethodType.ts we've started to call them "Gift card". I believe the reason why they're called "virtual cards" in the code is related to the fact that the ancestor of what we have today for virtual cards was called "gift cards", so the developers had to choose a different name. But now these old payment methods are all deleted and the public API could be a great place to start renaming them.

I therefore suggest:
```suggestion
      giftCardEmitterAccount: {
```

Though I'd be ok with just:
```suggestion
      giftCardEmitter: {
```

@znarf wdyt?

Reply:
@Betree 100% agree, "virtual card" has no place in the API v2, it should be "Gift Card" all the way.